examples/text/text_framerate_independent.c

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************************
+*
+*   raylib [core] example - Backspace input
+*
+*   Example demonstrates deletion of characters in a string independent of frame rate
+*
+*   Example originally created with raylib 5.0, last time updated with raylib 5.0
+*
+*   Example contributed by benevo (@benev0) and reviewed by Ramon Santamaria (@raysan5)
+*
+*   Example licensed under an unmodified zlib/libpng license, which is an OSI-certified,
+*   BSD-like license that allows static linking with closed source software
+*
+*   Copyright (c) 2023 benevo (@benev0)
+*
+********************************************************************************************/
+
+#include "raylib.h"
+
+#define MAX_INPUT_CHARS 50
+
+//------------------------------------------------------------------------------------
+// Program main entry point
+//------------------------------------------------------------------------------------
+int main(void)
+{
+    // Initialization
+    //--------------------------------------------------------------------------------------
+    const int screenWidth = 800;
+    const int screenHeight = 450;
+
+    InitWindow(screenWidth, screenHeight, "raylib [text] example - backspace input");
+    SetTargetFPS(1);
```

Comment:
This is not a framerate independent implementation if the example fixes the framerate to 1. Actually, that's a really bad and not-real practize, I'm afraid it could confuse many users.